src/components/AdviceGeneratorApp/AdviceGeneratorApp.tsx

```diff
@@ -3,16 +3,17 @@ import { useFetchAdvices } from "../../helpers/useFetchAdvices"
 
 export const AdviceGeneratorApp = () => {
 
-  const { adviceState } = useFetchAdvices();
+  const { advice, error, isLoading } = useFetchAdvices();
 
```

Comment:
Con base a las propiedades propuestas, puedes hacer un render condicional:

```suggestion
    if(loading) // O alternativamente puedes hacer if(!advice)
      return <h1>Loading...</h1>
    
   if(error)
      return <h1>Network request failed</h1>
      
   return (
       <main>
      <h4>Advice #  {advice?.id}</h4>
      <h1>{advice?.advice}</h1>
      <img src="/assets/images/pattern-divider-desktop.svg" alt="divider" />
      <img src="/assets/images/icon-dice.svg" alt="icon-dice" />
    </main>
   )
```

Reply:
El render se ha actualizado para mejorar la lectura del componente y ya se agregó la funcionalidad del botón para actualizar los consejos. 📖